src/client/components/charts/BellCurveChart.js

```diff
@@ -63,6 +74,7 @@ class BellCurveChart extends React.Component {
             title: '' // No title needed, overrides default
           }}
         />
+        <ExportButton getChartRef={this.getChartRef} />
```

Comment:
Should I keep the export button seperate and include it from the study files, in a similar fashion to how we deal with the download csv button?